frontend/restInterfaces/AccountRestInterface.ts

```diff
@@ -0,0 +1,79 @@
+import { BaseRestInterface } from './BaseRestInterface';
+import { IUser } from '../../model/user/IUser';
+
+/**
+ * Implements communication with the Server concerning all Account-requests
+ */
+export class AccountRestInterface extends BaseRestInterface {
+  /**
+   * Sends Login-Data to backend and validates them
+   * @param usernameOrEmail the username or email used to attempt to log in
+   * @param password the password used to attempt to log in
+   * @returns true if login was successfull
+   */
+  public static async verifyLogin(
```

Comment:
One small addition, this function should get a third parameter called `stayLoggedIn: boolean` which just gets passed on in the request